File: Design Pattern & Principles/AdapterPatternExample/PayPalAdapter.java
public class PayPalAdapter implements PaymentProcessor {
    private PayPalGateway payPal;

    @Override
    public void processPayment(double amount) {
        payPal.sendPayment(amount);
    }

    public PayPalAdapter(PayPalGateway payPal) {
        this.payPal = payPal;
    }

}

File: Week - 01 HandsOn/Design Principles & Patterns/AdapterPatternExample/PayPalAdapter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AdapterPatternExample
{
    public class PayPalAdapter : PaymentProcessor
    {
        private readonly PayPalGateway _payPal;

        public PayPalAdapter(PayPalGateway payPal)
        {
            _payPal = payPal;
        }

        public void ProcessPayment(string customerName, double amount)
        {
            _payPal.MakePayment(customerName, amount);
        }
    }

}
